raqm.c

```diff
@@ -35,26 +35,24 @@
 typedef struct _FriBidiRunStruct FriBidiRun;
 struct _FriBidiRunStruct
 {
-  FriBidiRun *prev;
-  FriBidiRun *next;
+    FriBidiRun *prev;
+    FriBidiRun *next;
```

Comment:
I’d avoid the indentation and other stylistic changes in this code, it follows FriBiDi code style because it was written to be submitted to FriBiDi. Though I think it is OK to change it to follow the style of the rest of the code, but in a separate commit please.